Assets/Scripts/ConnectionManagement/ConnectionState/HostingState.cs

```diff
@@ -71,9 +71,23 @@ public override void OnClientDisconnect(ulong clientId)
 
         public override void OnUserRequestedShutdown()
```

Comment:
Should we add this to the index? Something along the lines of "disconnecting all clients with reason", something like that?